### PR TITLE
Updated API

### PR DESCRIPTION
**Pull request by druiz0992**

### CODE REVIEW

File: docs/developers/api.md [hunk 1]
```diff
@@ -9,102 +8,35 @@ Example of these apps include:
 * Exchange integrations
 
 
-The API end points are divided in three main groups:
+The documentation of the API can be found [here](http://167.71.59.190:8001/)
```
Comment (arnaubennassar) [inline docs/developers/api.md]:
```suggestion
The documentation of the API can be found [here](https://apidoc.hermez.network/)
```

File: docs/developers/api.md [hunk 2]
```diff
@@ -9,102 +8,35 @@ Example of these apps include:
...
-
-### GET /state
-Return information that represents the current state of the network. It also includes metrics and statistics.
-
-### GET /config
-Return constant configuration of the network.
 
-### GET /tokens
-Get information of the supported tokens in the Hermez network.
+## Account
```
Comment (arnaubennassar) [inline docs/developers/api.md]: The sections of the swagger will be changed soon to coordinator / explorer endpoints: https://github.com/hermeznetwork/hermez-node/issues/409

**Review by arnaubennassar — CHANGES_REQUESTED** [review]

File: docs/developers/api.md [hunk 3]
```diff
@@ -9,102 +8,35 @@ Example of these apps include:
 * Exchange integrations
 
 
-The API end points are divided in three main groups:
+The documentation of the API can be found [here](http://167.71.59.190:8001/)
+
+In this section we will briefly outline the type of operations that can be performed with the API. In-depth information can be found in the link above.
```
Comment (jleebee) [inline docs/developers/api.md]: "outline the types of operations"

File: docs/developers/api.md [hunk 4]
```diff
@@ -9,102 +8,35 @@ Example of these apps include:
...
-### GET /tokens
-Get information of the supported tokens in the Hermez network.
+## Account
+This group of endpoints provides `account` and `exits` related information.
 
-### GET /tokens/{id}
-Get information of a token supported by Hermez network.
+There are two sets of `account` operations that can be performed:
+- Retrieving information from existing accounts, using `/accounts` endpoints.
+- Retrieving/configuring account authorizations using `/account-creation-authorization` endpoints. It is possible that a user authorizes the Coordinator the creation of an account with an Ethereum address using `POST /account-creation-authorization` endpoint. Then, the Coodinator will create the account at the moment this account receives a `Transfer`.
```
Comment (jleebee) [inline docs/developers/api.md]: "...that a user authorizes the Coordinator to create an account with an Ethereum address..."

File: docs/developers/api.md [hunk 5]
```diff
@@ -9,102 +8,35 @@ Example of these apps include:
...
 
-### GET /coordinators
-Get information about coordinators.
+Additionally, `exit` endpoints collect information necessary to perform a `Withdrawal` (such as the `balance`, the `siblings` and the `batch number`).
 
-### GET /coordinators/{bidderAddr}
-Get the information of a coordinator.
+## Transaction
+This group of endpoints provides `transaction` related information from the different databases. `Pool` database gathers all L2 transactions collected by a coordinator that are pending to be forged. `History` database gathers all (L1 and L2) forged transactions.
```
Comment (jleebee) [inline docs/developers/api.md]: "...that are pending forging."

**Review by jleebee — CHANGES_REQUESTED** [review]

File: docs/developers/api.md (hunk 2 above)
Comment (druiz0992) [reply docs/developers/api.md:21]: what will the coordinator endpoints be? 
POST/GET/ account-creation-authorization
POST /transaction-pool

Anything else?

**Comment by druiz0992** [comment]
I just followed Arnau`s offline suggestion and removed all extra text. I just left the link to the API.

**Review by arnaubennassar — APPROVED** [review]

**Pull request merged**